projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -328,6 +328,20 @@ public function get_upsell_nudge() {
 		}
 	}
 
+	/**
+	 * Adds Jetpack menu.
+	 *
+	 * @param int  $position The position of the menu item in the menu.
+	 * @param bool $seperator Whether to add a separator before the menu item.
+	 */
+	public function add_jetpack_menu( $position = 2, $seperator = false ) {
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
```

Comment:
This doesn't seem to be working, but I'm also wondering if this is a good approach.

Reply:
Seems like something was broken with my test site: p1706004199390829/1706004045.925339-slack-C06DN6QQVAQ